features/adServerKvps.md

```diff
@@ -0,0 +1,171 @@
+- **Reporting**. Some publishers rely on ad server key-values for important business reporting. The keys used for reporting could be any of the above, along with hb_source.
+
+## Obtaining Auction Results
+
+### Display and Native
+
+In the early versions of Prebid.js, there were a couple of basic functions
+publishers could use to get the 
+- [pbjs.setTargetingForGPTAsync](/dev-docs/publisher-api-reference/setTargetingForGPTAsync.html) - matches Google Publisher Toolkit ad slots to Prebid.js AdUnits, obtains the auction results for that adunit, and adds "targeting" values using GPT-provided functions.
+- [pbjs.getAdserverTargeting](/dev-docs/publisher-api-reference/getAdserverTargeting.html) - a more generic interface for obtaining KVPs
+
+All of the other functions available in the [publisher API](/dev-docs/publisher-api-reference.html) for obtaining auction bids came later.
+
+When there are a lot of adunits and bidders on a page, the number of KVPs being sent
+to the ad server can grow pretty large, so it quickly became apparent that many options were needed for controlling which KVPs these functions returned.
+
+Note that in old versions of Prebid.js, native ad components were passed via ad server KVPs.
+That approach has been deprecated -- all implementations should now use [one of the recommended approaches for native](/prebid/native-implementation.html).
+
+### Video
+
+Video's always been a different implementation than banners because
+it's the video player that controls the ad call, not in-page javascript like
+the GPT library. So the [Google Ad Manager Video module](/dev-docs/modules/dfp_video.html) includes the [buildVideoUrl](/dev-docs/publisher-api-reference/adServers.dfp.buildVideoUrl.html) function.
+
+Publishers using other ad servers need to integrate on their own
+using the [pbjs.getAdserverTargetingForAdUnitCode](/dev-docs/publisher-api-reference/getAdserverTargetingForAdUnitCode.html) function to build whatever
+needed to pass to the video player.
+
+## Controls
+
+Over the years, quite a few options have been added to adjust the number of bids and the exact set of KVPs sent to the ad server. This is an overlapping-but-powerful set of controls. There are often
+multiple ways to implement the same requirements, and there's no "wrong"
+way to do it.
+
+The list is ordered by those functions that Prebid recommends starting with:
+
+1. [enableSendAllBids](/dev-docs/publisher-api-reference/setConfig.html#setConfig-Send-All-Bids) - the grandaddy of targeting options. If false, only the winning set of KVPs will be sent. We recommend leaving this to the default value of true so that all bidders are represented in the final decision and for detailed reporting in the ad server, but setting it to false (aka "Send Top Bid" mode) is the most dramatic way to minimize what's sent to the ad server.
```

Comment:
Grandaddy :joy: 

Is the default still the recommendation? From our experience this caused more issues than it solved. Most of the time we didn't use it and it caused ad requests being truncated, which lead to a small amount of wrong targeted line items.

Maybe add an anchor link to the example below that explains this recommendation?